Automattic/wp-calypso
# Domain Management i1 : Add/bread crumb bulk contact info

Closes https://github.com/Automattic/dotcom-forge/issues/3653
## Proposed Changes

* Add breadcrumb to bulk edit contact info

<img width="590" alt="image" src="https://github.com/Automattic/wp-calypso/assets/47489215/21edff83-3c11-47a6-a825-4c4f252fd96c">


## Testing Instructions


*browse `/domains/manage?flags=domains/management`
* select domain and edit contact information
* verify All Domains > edit contact information breadcrumb is present 
* Verify clicking All Domains goes back to the All Domains table

## Pre-merge Checklist
